action.yml

```diff
@@ -0,0 +1,105 @@
+name: Dispatch CI
+description: GitHub Action to trigger ci.yml workflow_dispatch event via GitHub API
+
+runs:
+  using: composite
+  steps:
+
+    - name: Get type
+      id: gettype
+      shell: bash
+      env:
+        GITHUB_REPOSITORY: ${{ github.repository }}
+      run: |
+        # repository that has a twice-weekly schedule run on consequtive days - use day of week
+        date_part=$(date +%u)
+        type=""
+        if [[ $GITHUB_REPOSITORY == "silverstripe/installer" ]] || \
+            [[ $GITHUB_REPOSITORY == "silverstripe/recipe-kitchen-sink" ]]
+        then
+          # respository that has daily schedule - use hour
+          date_part=$(date +%H)
+        fi
+        if (( $date_part % 2 == 1 )); then
+          # latest major, next-minor e.g. "5"
+          type=latest,next-minor
+        else
+          # previous major, next-patch e.g. "4.13"
+          type=previous,next-patch
+        fi
```

Comment:
It's not super clear what the matrix is here... can you please clarify? Is it _only ever_ going to do `4.13` and `5`? Or will `5.0` be done sometimes? Will `4` ever be done?

Reply:
The matrix will be ["4.13",  "5"] .. unless we release 4.14 in which case it's ["4.14", "5"]

It'll also auto update to ["5.x", "6"] when 6 branches exist